### PR TITLE
Prepare 0.5.0 release

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by gedw99** [comment]
@mit-mit what does "The package now exclusively support Dart" mean. It always was for Dart or do you mean that its not interoperable with other GRPC implemenations. Would really like the back story on this as i heavily depend on using GRPC Dart with GRPC Golang and Web

**Comment by mit-mit** [comment]
See latest update, I was meant to say 'Dart 2'

**Comment by gedw99** [comment]
@mit-mit ok ... all good :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-## 0.5.0 - 2018-06-29
+## 0.5.0 - 2018-06-10
+
+* Breaking change: The package now exclusively supports Dart 2.
+* Fixed tests to pass in Dart 2.
+* Added support for Interceptors ([issue #79](https://github.com/grpc/grpc-dart/issues/79)); thanks to [@mogol](https://github.com/mogol) for contributing!
 
 * Implemented interceptors.
```
Comment (mogol) [inline CHANGELOG.md]: Should this line be removed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-## 0.5.0 - 2018-06-29
+## 0.5.0 - 2018-06-10
```
Comment (mogol) [inline CHANGELOG.md]: 2018-07-10 ?

Comment (mit-mit) [reply CHANGELOG.md]: Thanks @mogol, fixed!

**Pull request merged**